### PR TITLE
Add SIG docs: New Contributors Meet and Greet

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Adds the SIG docs: New Contributors Meet and Greet events as suggested by @sftim

### CODE REVIEW

**Comment by sftim** [comment]
Thanks!

/ok-to-test

**Review by sftim — CHANGES_REQUESTED** [review]
Instead of changing this file, change the [source](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1469), commit that change. Next, then run `make generate-dockerized` and commit the changes you get.

See https://github.com/kubernetes/community/blob/master/generator/README.md#how-to-use for more tips.

File: sig-docs/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ The [charter](charter.md) defines the scope and governance of the Docs Special I
 * Localization Subgroup Meeting: [Mondays at 15:00 UTC](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/).
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8).
+* New Contributors Meet and Greet: [Tuesdays at 10:30 UTC](https://zoom.us/j/95949750563?pwd=Rm51bGd0WFVQc2tPQWY3bEVuL0Fzdz09) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:30&tz=UTC).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/).
```
Comment (divya-mohan0209) [inline sig-docs/README.md:27]:
```suggestion
  * [Meeting notes and Agenda](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/).
  * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8)
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I would consider adding the Youtube link for the meeting recordings since they'll be available. You'll, of course, need to make changes to the sigs.yaml file again.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

File: sigs.yaml [hunk 2]
```diff
@@ -1507,6 +1507,14 @@ sigs:
     url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     archive_url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8
+  - description: New Contributors Meet and Greet
+    day: Tuesday
+    time: "10:30"
+    tz: UTC
+    frequency: monthly
+    url: https://zoom.us/j/95949750563?pwd=Rm51bGd0WFVQc2tPQWY3bEVuL0Fzdz09
```
Comment (divya-mohan0209) [inline sigs.yaml:1515]: I think this should be the below value per the calendar invite

```suggestion
    url: https://zoom.us/j/92822621820?pwd=S1p4RnB1RmNSZ3JjYlRRUVd4UjFPZz09
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Zoom URL has been updated after the first iteration. Please change it accordingly.

File: sigs.yaml (hunk 2 above)
Comment (Rishit-dagli) [reply sigs.yaml]: Ah, yes

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by Rishit-dagli** [comment]
The merge conflicts are now fixed!

> To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign nikhita after the PR has been reviewed.

/assign @nikhita

File: sigs.yaml [hunk 3]
```diff
@@ -1507,6 +1507,14 @@ sigs:
     url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     archive_url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8
+  - description: New Contributors Meet and Greet
+    day: Tuesday
+    time: "10:30"
+    tz: UTC
+    frequency: monthly
```
Comment (reylejano) [inline sigs.yaml:1514]:
```suggestion
    frequency: First Tuesday of the month
```

File: sig-list.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
...* [Natali Vlatko](https://github.com/natalisucks), Wayfair<br>* [Rey Lejano](https://github.com/reylejano), SUSE<br>|* [Slack](https://kubernetes.slack.com/messages/sig-docs)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)|* APAC SIG Meeting: [Wednesdays at 05:30 UTC (monthly - Wednesday, after the fourth Tuesday, every month)](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit)<br>* Korean Team Meeting: [Thursdays at 13:00 UTC (biweekly)](https://docs.google.com/document/d/1h5sMhBpPB5unJmBAS7KzDiPs-_eFQOu5o4UyHwMtFCA/edit)<br>* Localization Subgroup Meeting: [Mondays at 15:00 UTC (monthly)](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/)<br>* New Contributors Meet and Greet: [Tuesdays at 10:30 UTC (monthly)](https://zoom.us/j/92822621820?pwd=S1p4RnB1RmNSZ3JjYlRRUVd4UjFPZz09)<br>* Regular SIG Meeting: [Tuesdays at 17:30 UTC (weekly - except fourth Tuesday every month)](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit)<br>* Spanish Team Meeting: [Tuesdays at 15:30 UTC (weekly)](https://zoom.us/j/95918289494?pwd=Wk9Oa0xZUkFXSDV5OTFoZEZsTURCZz09)<br>
```
Comment (reylejano) [inline sig-list.md:44]:
```suggestion
|[Docs](sig-docs/README.md)|[docs](https://github.com/kubernetes/kubernetes/labels/sig%2Fdocs)|* [Divya Mohan](https://github.com/divya-mohan0209), SUSE<br>* [Jim Angel](https://github.com/jimangel), Google<br>* [Natali Vlatko](https://github.com/natalisucks), Wayfair<br>* [Rey Lejano](https://github.com/reylejano), SUSE<br>|* [Slack](https://kubernetes.slack.com/messages/sig-docs)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)|* APAC SIG Meeting: [Wednesdays at 05:30 UTC (monthly - Wednesday, after the fourth Tuesday, every month)](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit)<br>* Korean Team Meeting: [Thursdays at 13:00 UTC (biweekly)](https://docs.google.com/document/d/1h5sMhBpPB5unJmBAS7KzDiPs-_eFQOu5o4UyHwMtFCA/edit)<br>* Localization Subgroup Meeting: [Mondays at 15:00 UTC (first Monday of the month)](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/)<br>* New Contributors Meet and Greet: [Tuesdays at 10:30 UTC (first Tuesday of the month)](https://zoom.us/j/92822621820?pwd=S1p4RnB1RmNSZ3JjYlRRUVd4UjFPZz09)<br>* Regular SIG Meeting: [Tuesdays at 17:30 UTC (weekly - except fourth Tuesday every month)](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit)<br>* Spanish Team Meeting: [Tuesdays at 15:30 UTC (weekly)](https://zoom.us/j/95918289494?pwd=Wk9Oa0xZUkFXSDV5OTFoZEZsTURCZz09)<br>
```

File: sig-docs/README.md [hunk 5]
```diff
@@ -23,6 +23,9 @@ The [charter](charter.md) defines the scope and governance of the Docs Special I
 * Localization Subgroup Meeting: [Mondays at 15:00 UTC](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/).
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8).
+* New Contributors Meet and Greet: [Tuesdays at 10:30 UTC](https://zoom.us/j/92822621820?pwd=S1p4RnB1RmNSZ3JjYlRRUVd4UjFPZz09) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:30&tz=UTC).
```
Comment (reylejano) [inline sig-docs/README.md:26]:
```suggestion
* New Contributor Meet and Greet: [Tuesdays at 10:30 UTC](https://zoom.us/j/92822621820?pwd=S1p4RnB1RmNSZ3JjYlRRUVd4UjFPZz09) (first Tuesday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:30&tz=UTC).
```

File: sigs.yaml [hunk 6]
```diff
@@ -1507,6 +1507,14 @@ sigs:
     url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     archive_url: https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8
+  - description: New Contributors Meet and Greet
```
Comment (reylejano) [inline sigs.yaml:1510]:
```suggestion
  - description: New Contributor Meet and Greet
```

**Comment by Rishit-dagli** [comment]
@reylejano https://github.com/kubernetes/community/pull/6679/commits/627aed33d22320445abe265f7c688ed64e334883 and https://github.com/kubernetes/community/pull/6679/commits/379fbe3c227a4f5e87bf4273c374eb2a1e8e42ee address your reviews

**Comment by reylejano** [comment]
/lgtm
lgtm's from 2 SIG Docs co-chairs (myself and Divya) and 1 SIG Docs tech lead (Tim)
/assign @nikhita

**Review by mrbobbytables — COMMENTED** [review]
/approve

I've added the tide squash label, but please rebase and fixup and PRs with the WIP type commits in the future. It's less critical here in k/community, but can be a blocker in other repos.

**Pull request merged**